actioncable/blade.yml

```diff
@@ -5,3 +5,9 @@ load_paths:
 
 logical_paths:
   - test.js
+
+build:
+  logical_paths:
+    - action_cable.js
+  path: lib/assets/compiled
+  js_compressor: uglifier
```

Comment:
For that kind of hackery, you could point to the uncompiled `action_cable.coffee` for inserting your `console.log`s, etc.

>  especially as it doesn't really matter -- it'll be compressed eventually either way

That's true if your app compresses assets, but it's not a given. Especially considering people can install via npm and use `action_cable.js` outside of Rails / Sprockets.

I don't feel strongly that it should be compressed given its relatively small file size, but it feels like the right thing to do for a distributed .js file. FWIW, Turbolinks is compressed using the same setup.